### ISSUE TITLE
Glover crash

### ISSUE THREAD

**Issue by fzurita** [mupen64plus/mupen64plus-core#171]
Glover is crashing with the new arm dynarec. It crashes within a few seconds of starting the game. The core crashes and produces this core dump:

```
Build fingerprint: 'motorola/quark_verizon/quark:5.1/SU4TL-49/5:user/release-keys'
pid: 14559, tid: 30188, name: CoreThread  >>> org.mupen64plusae.v3.alpha <<<
signal 4 (SIGILL), code 1 (ILL_ILLOPC), fault addr 0x8f5e5560
Stack frame 07-29 02:10:07.638 400-400/? I/DEBUG:     #00 pc 001f2560  <unknown>: Unable to open symbol file .\Mupen64Plus-AE-Studio\mobile\src\main\obj\local\armeabi-v7a/<unknown>. Error (22): Invalid argument
Stack frame 07-29 02:10:07.638 400-400/? I/DEBUG:     #01 pc fffffffd  <unknown>: Unable to open symbol file .\Mupen64Plus-AE-Studio\mobile\src\main\obj\local\armeabi-v7a/<unknown>. Error (22): Invalid argument
```

This is probably the relevant portion:

**signal 4 (SIGILL), code 1 (ILL_ILLOPC), fault addr 0x8f5e5560**

**Comment by fzurita** [comment]
@Gillou68310 Did you see my other issue as well?

https://github.com/mupen64plus/mupen64plus-core/issues/170

**Comment by fzurita** [comment]
Here is another crash: https://github.com/mupen64plus/mupen64plus-core/issues/158

**Comment by fzurita** [comment]
Also, I just discovered that the new_dynarec (arm32 version) Also has the same problem as the arm64 version in CBFD. When loading a save state, some TLB check assertion fails.

**Comment by Gillou68310** [comment]
I guess this is the same issue as https://github.com/mupen64plus-ae/mupen64plus-ae/issues/408
I'm still not happy with the fix I provided but I guess we can temporary merge it until I find the root cause of the issue.

**Comment by Gillou68310** [comment]
@Narann could also assign me to this issue so I won't forget about it ;-)

**Comment by Gillou68310** [comment]
@fzurita this should be fixed by #871?

**Comment by fzurita** [comment]
Yep, fixed! We can close it once it's merged.
